bin/cron/update_dotd

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env ruby
+require 'open-uri'
+require 'json'
+require_relative '../../deployment'
+
+slack_token = CDO.slack_token
+pagerduty_token = CDO.pagerduty_token
+channel_id = CDO.developers_channel_id
+
+# Get current developers room topic
+channel_info = open("https://slack.com/api/channels.info?token=#{slack_token}&channel=#{channel_id}").read
+current_topic = JSON.parse(channel_info)["channel"]["topic"]["value"]
+
+# Get DOTD on call from Pager Duty
+on_call = `curl -H "Content-type: application/json" -H "Authorization: Token token=#{pagerduty_token}" -X GET -G "https://codeorg.pagerduty.com/api/v1/users/on_call"`
+dotd_email = JSON.parse(on_call)["users"].first["email"]
+
+# Match Pager Duty email to Slack mention name
+mention_name = "DNE"
+users_list = open("https://slack.com/api/users.list?token=#{slack_token}").read
+JSON.parse(users_list)["members"].each { |member| mention_name = member["name"] if dotd_email.eql? member["profile"]["email"] }
```

Comment:
I could explain it more and update the topic to something like "Couldn't identify DotD, please check schedule."